UniquePtrTest.cpp

```diff
@@ -0,0 +1,11 @@
+#include "TestHarness.h"
+
+TEST(unique_ptrTest, int)
+{
+    auto intValues = std::make_unique<int[]>(10);
+    for (auto index = 0; index < 10; index++)
+    {
+        intValues[index] = index;
+        CHECK_EQUAL(index, intValues[index]);
```

Comment:
OK, but it's a good practice to completely finish your setup and modifications of the Thing Being Tested, before doing your tests on it.  So that would look like 2 loops here.